test/integration/transaction.test.js

```diff
@@ -79,6 +79,32 @@ if (current.dialect.supports.transactions) {
         });
       });
 
+      it('supports running hooks when a transaction is commited', function() {
+        const self = this;
+        const hook = sinon.spy();
+        let t;
+        return expect(this.sequelize.transaction(transaction => {
+          t = transaction;
+          transaction.afterCommit(hook);
+          return self.sequelize.query('SELECT 1+1', {transaction, raw: true});
```

Comment:
No need to pass ``{ raw: true }``, its not required. Instead pass `type: QueryType.SELECT`